lms/djangoapps/grades/new/course_grade.py

```diff
@@ -26,6 +26,12 @@ def __init__(self, student, course, course_structure):
         self.course_structure = course_structure
         self.chapter_grades = []
 
+        # TODO: currently, data model won't accept null for course version, but tests do that a lot. Determine correct solution to this problem.
+            # - course_version not always needed for PersistentSubsectionGrade?
+            # - tests all need to ensure their courses have a course_version?
+        if self.course.course_version is None:
+            self.course.course_version = 1
```

Comment:
This needs to be fleshed out further. Do we expect every course we get passed to contain `course_version`, so that we ought to fail on that condition (and update calling tests to actually do that)? Or should we make the data model robust to missing that value?